wordpress-mobile/WordPress-iOS
# Stats: Total Subscribers Card

Fixes #23048

<img width="374" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/a3a6eb10-4a29-4ec3-b8da-94e384f16967">

## Solution

1. Updated `subscribersList` to return both subscribers and subscribers totals since they come from the same endpoint
2. Added Total Subscribers card, reusing the same UI from Insights

Note: Insights Totals Subscribers card may show different numbers since it also adds publicize numbers. Fix is made here https://github.com/wordpress-mobile/WordPress-iOS/pull/23099

## To test:

1. Enable Stats Traffic and Subscribers tab feature flag
2. Open Stats -> Subscribers
3. Confirm Total Subscribers card appears
4. Compare the number with the web

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

Added unit tests

PR submission checklist:


Testing checklist:
